samples/net/wpanusb/overlay-mcr20a.conf

```diff
@@ -0,0 +1,6 @@
+CONFIG_GPIO=y
+CONFIG_SPI=y
+CONFIG_SPI_0=y
```

Comment:
The build for frdm_k64f will fail. With the consideration of e240773d166081aa1a18b2d3d4ba3645873d1622 the case as it is here remains untested. I recommend to change the example in the commit message from frdm_k64f to usb_kw24d512. The line 3 must then be removed.